### PR TITLE
Bst 2675

### PR DESCRIPTION
**Pull request by pavel-kokolemin**
http://185.213.153.35:8081/browse/BST-2675

### CODE REVIEW

File: BlocksettleNetworkingLib/ProtobufHeadlessUtils.cpp [hunk 1]
```diff
@@ -85,8 +87,12 @@ bs::core::wallet::TXSignRequest bs::signer::pbTxRequestToCore(const headless::Si
          txSignReq.inputs.push_back(utxo);
       }
    }
-   txSignReq.inputIndices.insert(txSignReq.inputIndices.end()
-      , request.input_indices().begin(), request.input_indices().end());
+   for (const auto &inputIndex : request.input_indices()) {
+      if (!inputIndex.empty()) {
+         bs::hd::Path::fromString(inputIndex);
```
Comment (sergey-chernikov) [inline BlocksettleNetworkingLib/ProtobufHeadlessUtils.cpp:92]: Where does the return value of this function go to?

File: BlocksettleNetworkingLib/ProtobufHeadlessUtils.cpp [hunk 2]
```diff
@@ -103,6 +109,7 @@ bs::core::wallet::TXSignRequest bs::signer::pbTxRequestToCore(const headless::Si
    }
 
    if (request.has_change()) {
+      bs::hd::Path::fromString(request.change().index());
```
Comment (sergey-chernikov) [inline BlocksettleNetworkingLib/ProtobufHeadlessUtils.cpp:112]: Same here - is it just dropped? Why at least the result is not checked for emptyness / valid number of path elements?

File: WalletsLib/CoreWallet.cpp [hunk 3]
```diff
@@ -658,38 +658,6 @@ bool wallet::TXSignRequest::isSourceOfTx(const Tx &signedTx) const
    }
 }
 
-bool wallet::TXSignRequest::populateSupportingTx(std::shared_ptr<ArmoryConnection> connPtr)
```
Comment (sergey-chernikov) [inline WalletsLib/CoreWallet.cpp:661]: Didn't find a replacement for this removed code. Is it just vanished?

File: BlocksettleNetworkingLib/ProtobufHeadlessUtils.cpp (hunk 1 above)
Comment (pavel-kokolemin) [reply BlocksettleNetworkingLib/ProtobufHeadlessUtils.cpp]: Sure, let's check path length.

File: BlocksettleNetworkingLib/ProtobufHeadlessUtils.cpp (hunk 2 above)
Comment (pavel-kokolemin) [reply BlocksettleNetworkingLib/ProtobufHeadlessUtils.cpp]: Fixed.

File: WalletsLib/CoreWallet.cpp (hunk 3 above)
Comment (pavel-kokolemin) [reply WalletsLib/CoreWallet.cpp:661]: Moved out earlier to `CreateTransactionDialog::CreateTransaction`

**Pull request merged**